Validation/RecoTrack/python/plotting/trackingPlots.py

```diff
@@ -424,7 +426,8 @@ def _makeMVAPlots(num, hp=False):
                               _makeDistPlots("vertpos", "ref. point r (cm)", common=dict(xlog=True)) +
                               _makeDistPlots("zpos"   , "ref. point z (cm)") +
                               _makeDistPlots("simpvz" , "Sim. PV z (cm)", common=dict(xmin=_minZ, xmax=_maxZ)) +
-                              _makeDistPlots("dr"     , "min #DeltaR", common=dict(xlog=True)),
+                              _makeDistPlots("dr"     , "min #DeltaR", common=dict(xlog=True)) +
+                              _makeDistPlots("drj"     , "#DeltaR(track, jet)", common=dict(xlog=True)),
                               ncols=4, legendDy=_legendDy_4rows,
```

Comment:
So about the splitting of plots, please split this one to two:
*  `_extDistPos = PlotGroup("distPos",` containing `vertpos`, `zpos`, and `simpvz` and leave the `legendDy` parameter out
* `_extDistDeltaR = PlotGroup("distDeltaR",` containing `dr` and `drj` with `legendDy=_legendDy_2rows`

Reply:
done